### DISCUSSION TITLE
Unrecognised option "--active_beacon"

### DISCUSSION THREAD

**Discussion by hyekalhitech** [ZerBea/hcxdumptool#384] [Q&A]
Hi, anyone can help? Why does the hcxdumptool was unable to recognize my --active_beacon" & my --enable_status command ? Its weird. ![IMG_20231217_015101.jpg](https://github.com/ZerBea/hcxdumptool/assets/44819927/c73e5165-49d4-4279-bfd7-12f4afbe34f8)

**Comment by ZerBea** [comment]
The first step should always be to read the help:
```
$ hcxdumptool -h
$ hcxdumptool --help

```
directly followed by README.md and changelog.

Everything is explained there.
